shared-ui/components/secondary-button/SecondaryButtonWhiteText.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { StyledSecondaryButtonWhiteText } from './SecondaryButton.styles';
+import { ButtonProps } from '../../lib/types';
+
+const SecondaryButtonWhiteText: React.FC<ButtonProps> = ({ btnText, btnLink }) => {
+  return (
+    <a href={btnLink}>
+      <StyledSecondaryButtonWhiteText>
```

Comment:
Same here - lets remove this file and pass in a prop so we can use the same button to handle both styles